src/ResourceManager/Network/Commands.Network/ApplicationGateway/Probe/AzureApplicationGatewayProbeConfigBase.cs

```diff
@@ -62,6 +63,22 @@ public class AzureApplicationGatewayProbeConfigBase : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public uint UnhealthyThreshold { get; set; }
 
+        [Parameter(
+           Mandatory = false,
+           HelpMessage = "Whether the host header should be picked from the backend http settings. Default value is false")]
+        public SwitchParameter PickHostNameFromBackendHttpSettings { get; set; }
+
+        [Parameter(
+           Mandatory = false,
+           HelpMessage = "Minimum number of servers that are always marked healthy. Default value is 0")]
+        public uint MinServers { get; set; }
```

Comment:
@akshaysngupta you can use the `ValidateRange` attribute to restrict the values passed through to this parameter instead of using the `uint` type. For example:

```cs
[Parameter(
    Mandatory = false,
    HelpMessage = "Whether the host header should be picked from the backend http settings. Default value is false")]
[ValidateRange(0, int.MaxValue)]
public int MinServers { get; set; }
```